docs/Getting-Started.md

```diff
@@ -0,0 +1,270 @@
+## Tux Getting Started Guide
+To get started hacking with `Tux`, let's see what some parts of a Todo component might look like inside a To-Do Application built with `Tux`.
+
+## Tux Environment Setup
+Prior to getting started on the Todo application using `Tux` we need setup our working environment. `Tux` is built with CommonJS and thus you will need a compiler such as `Browserify` or `webpack`. In our case, we use `Browserify` for compiling, `Reactify` for compiling JSX, `Envify` for accessing NODE_ENV variables in the browser (great for automatically turning dev tools on and off), and `Watchify` for automatic compiling.
+
+    npm install --save browserify
+    npm install --save envify
+    npm install --save reactify
+    npm install --save watchify
+
+To use these modules in development, we recommend adding the following lines to the `scripts` key of your `package.json` file to set up automatic JSX compiling and configure testing:
+
+    "scripts": {
+      "start": "watchify -d [YOUR MAIN.JS OR SIMILAR FILE] -o [YOUR OUTPUT BUNDLE.JS] -v",
+      "build": "NODE_ENV=production browserify [YOUR MAIN.JS OR SIMILAR FILE] | uglifyjs -cm > [YOUR OUTPUT BUNDLE.JS]"
+    }
+
+Include this `browserify` transform in your `package.json` file as well:
```

Comment:
Add sentence explaining: this tells `browserify` to include the `reactify` transform (with `es6` enabled for things like `spread` syntax) and the `envify` transform.